src/ParkDetails/ParkDetails.js

```diff
@@ -6,31 +6,73 @@ function ParkDetails({ foundPark }) {
   if (!foundPark) {
     return null;
   }
-  console.log(foundPark);
+
+  let parkImage;
+  if (foundPark.attributes.media && foundPark.attributes.media.length > 1) {
+    parkImage = (
+      <img
+        className="single-image"
+        src={foundPark.attributes.media[1].url}
+        alt={foundPark.attributes.media[1].altText}
+      />
+    );
+  } else {
+    parkImage = (
+      <img
+        className="single-image"
+        src={foundPark.attributes.media[0].url}
+        alt={foundPark.attributes.media[0].altText}
+      />
+    );
+  }
+
   return (
     <div className="park-details-container">
       <div className="park-details-title">
         <h2 className="single-title">{foundPark.attributes.name}</h2>
       </div>
-      <section className="image-container">
-        <img
-          className="single-image"
-          src={foundPark.attributes.media[1].url}
-          alt={foundPark.attributes.media[0].altText}
-        />
+      <section className="image-container">{parkImage}</section>
+      <section className="park-description">
+        <div className="single-park-description">
+          <h3>Park Info</h3>
+          <p>{foundPark.attributes.designation}</p>
+          <p>{foundPark.attributes.description}</p>
+        </div>
+      </section>
+      <section className="active-alerts">
+        <div className="single-park-alerts">
+          <h3>Active Alerts</h3>
+          {foundPark.attributes.active_alerts[0] ? (
+            <>
```

Comment:
i am blown away at hopw good this all looks and how well it flows! you knocked it out of the park with this formatting